README.md

```diff
@@ -12,9 +12,23 @@ For details of installation and general usage, visit the [d3fc project webpage](
 
 ## Developing
 
-[npm](https://www.npmjs.com/), the package manager for [Node.js](https://nodejs.org/), is used to manage the project's dependencies. The project is built and tested via npm scripts, with releases managed via [semantic release](https://github.com/semantic-release/semantic-release).
+This project takes a mono-repo approach, with [Lerna](https://lernajs.io/) managing the various packages and their dependencies. A typical development cycle is as follows:
 
-This project acts as a bundler for the individual d3fc packages, which are all listed under the [d3fc organization](https://github.com/d3fc). If you want to contribute features or fixes to d3fc you should locate the correct package and make your changes there.
+~~~
+npm install
+~~~
+
+This install dependencies, and runs `lerna bootstrap` in order to manage the cross-dependencies between the various projects.
+
+When making changes to a package, you can either execute `npm run bundle` within the package folder to build just that package, or you can run the following from the project root:
```

Comment:
Might be worth saying that you need to build the dependencies of that package first i.e. a  top level npm run test.

Reply:
I've resolved this by simply telling people to run `npm install` the `npm test` from the get-go